functions/HttpTriggerAPIUsersId/__init__.py

```diff
@@ -11,6 +12,13 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
     logging.info(
         'Python HTTP trigger for users/userId is processing a request ')
 
+    #Initiating REDIS cache
+    REDIS_HOST = 'nsc-redis-dev-usw2-thursday.redis.cache.windows.net'
```

Comment:
What if the host changes? This should be pulled from app configuration/environment